LEDWeather.py

```diff
@@ -28,31 +32,32 @@
 
 def connect():
     serialPorts = glob.glob("/dev/cu.usbmodem*")
-    port = serialPorts[0]
+    # port = serialPorts[0]
+    port = 'com3'
```

Comment:
accidental change?

Reply:
intentional as that system doesn't work on wiondows

Reply:
Well, now it only works on windows and only on com3, that's not an improvement I guess.

Reply:
also has nothing to do with PR scope.